.github/workflows/triage.yaml

```
@@ -0,0 +1,52 @@
+on:
+  issues:
+    types:
+    - opened
+    - reopened
+  pull_request:
+    types:
+    - opened
+    - reopened
+jobs:
+  issues:
```

Should this be added to reduce the permissions scope of the token?

```suggestion
  issues:
    permissions:
      issues: write
```

---

Good point; will fix :+1:.